doc/changes/package_diffs/2.0.0/diff_standard-EXASOL-7.1.0_standard-EXASOL-7.1.0/README.md

```diff
@@ -1,5 +1,9 @@
 # Package Version Comparison between Standard-EXASOL-7.1.0 flavor in 2.0.0 and Standard-EXASOL-7.1.0 flavor in 1.1.0
```

Comment:
Why do we have changes in this file?

Reply:
I realized the references to most of the diff files were missing, it had only references to python2/python3 diff files. I remember that version 2.0.0 had mostly changes on Python packages, and I believe this is the reason the other diff files were removed.